src/app/pages/wallets/my-wallets/wallet-form-modal/wallet-form-modal.component.ts

```diff
@@ -17,15 +17,24 @@ export class WalletFormModalComponent {
         @Inject(MAT_DIALOG_DATA) public data: MyWallet | undefined,
     ) {
         this.form = new FormGroup<IWalletModalFormType>({
-            name: new FormControl(this.data?.name ?? '', { nonNullable: true }),
+            name: new FormControl(this.data?.name ?? '', { validators: Validators.required, nonNullable: true }),
         });
     }
 
+    get formName() { return this.form.get('name'); }
+
+    public showNameError() {
+      return this.formName?.invalid && this.formName?.touched;
+    }
+
     public save(): void {
-        this.dialogRef.close(this.form.value);
+        if(this.form.valid) {
```

Comment:
Try to avoid nesting whenever possible, better form would be:
```typescript
if (this.form.invalid) {
    return;
}

this.dialogRef.close(this.form.value);
```